### ISSUE TITLE
Arduino code not compiling

### ISSUE THREAD

**Issue by engrfash** [EinfachArne/Somfy_Remote#1]
This is my first time tring this library out and i get this error from arduino when i wanna compile

In file included from C:\Users\Engr Fash\Documents\Arduino\libraries\Basecamp/Basecamp.hpp:10:0,

                 from C:\Users\Engr Fash\Documents\Arduino\libraries\Somfy\examples\WiFi_Multi_Remote_ESP32\WiFi_Multi_Remote_ESP32.ino:25:

C:\Users\Engr Fash\Documents\Arduino\libraries\Basecamp/Configuration.hpp:12:19: fatal error: sstream: No such file or directory

compilation terminated.

exit status 1
Error compiling for board Arduino Uno WiFi.

**Comment by EinfachArne** [comment]
Hi @engrfash,

thank you for your question. I just released the first productive version 1.0.0 of this library. In this release I scrapped the MQTT examples as MQTT is not part of my library and so the examples based on another third party library called [Basecamp](https://github.com/merlinschumacher/Basecamp). As this library is not being updated anymore for over a year now I decided to stick only with the 4 basic functional examples. 

If you want to assure that your setup works, just use on of the 4 examples provided.

I will showcase a possible integration with OpenHAB via the [Homie convention](https://github.com/homieiot/convention) on a different platform and add a link here.

**Comment by engrfash** [comment]
Hi EinfachArne,
You are a life saver.
Thanks so much for your library, I have ordered for the CC1101 but it is yet to arrive.

I still have few questions tho
1. I noticed there is no way to select which board you are using in the new code. Does it select automatically?
2. I'm so waiting for an MQTT support or Openhab support. Can't wait to start controlling m blinds through Alexa. Thanks Alot for this Library. Expecting your links
